tox.ini

```diff
@@ -28,20 +27,17 @@ max-doc-length = 120
 ; D413 = Missing blank line after last section (numpy style)
 ; D414 = Section has no content (numpy style)
 ignore = D101,D200,D203,D212,D215,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414
-match-dir = (?!migrations)
 
 [pytest]
 DJANGO_SETTINGS_MODULE = test_utils.test_settings
-addopts = --cov openedx_events --cov tests --cov-report term-missing --cov-report xml
+addopts = --cov openedx_events --cov-report term-missing --cov-report xml
 norecursedirs = .* docs requirements site-packages
 
 [testenv]
 deps =
-    django32: Django>=3.2,<4.0
     django42: Django>=4.2,<4.3
     -r{toxinidir}/requirements/test.txt
 commands =
-    python manage.py check
```

Comment:
it running `manage.py check` no longer part of the standard? I suppose this is copied from a template in cookicutters right? This perhaps https://github.com/openedx/edx-cookiecutters/blob/bdde9a06b5a0b3922ce4f7f92db4c3816fb94319/python-template/%7B%7Bcookiecutter.placeholder_repo_name%7D%7D/tox.ini#L34?
However I could not find when this was removed.

Reply:
Mmm, yes. I removed it because of it so I don't have a good argument in favor. I put it back so we can think about it later.